### PR TITLE
adding product by admin and notify to users

### LINKED ISSUE #19: ability to adding product by admin

### PR DESCRIPTION
**Pull request by gitygity**
fix #19
 
in this sprint :
- [ x ] adding admin in .env file
- [ x ] add product by admin
- [ x ] notify to users that they have registered in bot

### CODE REVIEW

File: gity_startingup2017_bot/git0bot.py [hunk 1]
```diff
@@ -16,160 +17,37 @@
                            ' %(levelname)s -'
                            ' %(message)s ',
                     level=logging.INFO)
-server_r = redis.Redis()
-isadmin = 0
+server_r = redis.Redis(decode_responses=True)
+
+# server_r.delete('order6')
```
Comment (ahangarha) [inline gity_startingup2017_bot/git0bot.py]: This must be removed

Comment (ahangarha) [inline gity_startingup2017_bot/git0bot.py:22]: This must be removed

File: gity_startingup2017_bot/git0bot.py [hunk 2]
```diff
@@ -16,160 +17,37 @@
...
+
+
+print(server_r.get('idp'))
```
Comment (ahangarha) [inline gity_startingup2017_bot/git0bot.py:25]: This also could be used during development. If no other information is going to be provieded, I think this line should be commented out.

File: gity_startingup2017_bot/git0bot.py (hunk 1 above)
Comment (gitygity) [reply gity_startingup2017_bot/git0bot.py:22]: Ok thx

File: gity_startingup2017_bot/git0bot.py (hunk 2 above)
Comment (gitygity) [reply gity_startingup2017_bot/git0bot.py:25]: Yes your right i forget remove test prints . I will change thank you for reviewing

**Review by ahangarha — APPROVED** [review]

**Pull request closed without merging**